.github/workflows/release.yml

```diff
@@ -114,16 +113,13 @@ jobs:
         # TODO: add back 'Windows' when it can be compiled.
         os: ['macOS', 'Linux']
         py-version: ['3.7', '3.8', '3.9']
-        tf-version: ['2.8.3']
-        tf-need-cuda: ['1', '0']
+        tf-need-cuda: ['0', '1']
         cpu: ['x86']
-#       TODO(poinwater): add macOS CI once GitHub supports macOS 12.0.0 + 
-#        include:
-#          - os: 'macOS'
-#            cpu: 'arm64'
-#            tf-version: '2.5.0'
-#            py-version: '3.9'
-#            tf-need-cuda: '0'
+        include:
+          - os: 'macos-12'
+            cpu: 'arm64'
+            tf-version: '2.9.0'
+            py-version: '3.9'
```

Comment:
Maybe miss the "tf-need-cuda: '0'"

Reply:
I wanna totally delete the MacOS Arm64 build from the PR, because the GitHub-hosted m1 runner is not ready.

Reply:
I will add a self-hosted m1 runner in the next PR. I am investigating the solution.

Reply:
OK